recipe/meta.yaml

```diff
@@ -57,8 +70,11 @@ requirements:
     - requests
     - h5py
     - {{ pin_compatible('numpy') }}
-    # cuda-version is added here to prevent the conda-forge runner to download a different version during testing, which makes it run out of memory. 
-    - cuda-version =={{ cuda_compiler_version }}  # [cuda_compiler_version not in (undefined, 'None')] 
+# cuda-version is added here to prevent the conda-forge runner to download a different version during testing, which makes it run out of memory.
```

Comment:
memory like ram or memory like disk space? There are some new settings that can help fix these issues (example https://github.com/regro-cf-autotick-bot/dgl-feedstock/blob/0c8ef3417f5ef5569be42e3b7a4ae52fef065b52/conda-forge.yml#L1-L4 )

you will need to re-render after you make those changes, do you think this is worth a shot?

Reply:
@RaulPPelaez sorry forgot to tag you, let me know what you think

Reply:
Disk space. Nice settings! Lets add them and try.